sys/fido2/ctap/ctap.c

```diff
@@ -47,16 +47,17 @@ typedef struct {
  * state of authenticator. Stored in flash memory
  */
 typedef struct {
-    uint8_t initialized;                        /**< CTAP is initialized or not */
+    uint8_t initialized_marker;                 /**< CTAP initialized marker */
     bool pin_is_set;                            /**< PIN is set or not */
     int rem_pin_att;                            /**< remaining PIN tries */
     uint8_t pin_hash[SHA256_DIGEST_LENGTH / 2]; /**< LEFT(SHA-256(pin), 16) */
     ctap_config_t config;                       /**< configuration of authenticator */
     uint16_t rk_amount_stored;                  /**< total number of resident keys stored on device */
     uint8_t cred_key[CTAP_CRED_KEY_LEN];        /**< AES CCM encryption key for cred */
+    bool cred_key_is_initialized;               /**< AES CCM key initialized flag */
 } ctap_state_t;
```

Comment:
Just realized: is it on purpose to define this struct in the c file and not the header?

Reply:
Yes it is, since it is only used in the c file.